app/api/business/validators/application_validator.py

```diff
@@ -212,12 +212,46 @@ def validate_methods(self):
 
     def validate_recruiter(self):
         errors = []
-        if not self.application.data.get('recruiter'):
+        recruiter = self.application.data.get('recruiter')
+
+        if not recruiter:
             errors.append({
                 'message': 'Recruiter is required',
                 'severity': 'error',
                 'step': 'recruiter'
             })
+
+        if recruiter == 'yes' or recruiter == 'both':
+            labour_hire = self.application.data.get('labourHire', {})
+            now = pendulum.now('Australia/Canberra').date()
+            for state, state_value in labour_hire.iteritems():
+                if not state_value:
+                    continue
+                licence_number = state_value.get('licenceNumber')
+                expiry = state_value.get('expiry')
+                if (licence_number and not expiry) or (not licence_number and expiry):
+                    errors.append({
+                        'message': 'Licence number and expiry must be both filled for {}'.format(state.upper()),
+                        'severity': 'error',
+                        'step': 'recruiter'
+                    })
+
+                if expiry:
+                    try:
+                        expiry_date = pendulum.parse(expiry, tz='Australia/Sydney')
+
+                        if now > expiry_date.date():
```

Comment:
Should we check for dates that don't exist in the future? Like 31st June or dates during leap year